src/material-examples/button-toggle-exclusive/button-toggle-exclusive-example.html

```diff
@@ -1,14 +1,14 @@
 <mat-button-toggle-group #group="matButtonToggleGroup">
-  <mat-button-toggle value="left">
+  <mat-button-toggle value="left" aria-label="left">
```

Comment:
I'm not sure that these aria labels are very useful. How about "Text align left"?

Reply:
Yes, the label should be more human readable. I would go with "Align text left", etc.

Reply:
"Text align left" etc. is better, as we have "justify" option. "Text align justify" looks better than "Align text justify" something.